src/SFA.DAS.AANHub.Application/Apprentices/Commands/PatchApprenticeMember/PatchApprenticeMemberCommandValidator.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.ObjectModel;
+using FluentValidation;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Application.Apprentices.Commands.PatchApprenticeMember
+{
+    public class PatchApprenticeMemberCommandValidator : AbstractValidator<PatchApprenticeMemberCommand>
+    {
+        public const string Replace = "replace";
+        public const string NoPatchOperationsPresentErrorMessage = "There are no patch operations in this call";
+        public const string PatchOperationsLimitExceededErrorMessage = "The number of patch operations in this call exceed the expected count";
+        public const string PatchOperationContainsUnavailableFieldErrorMessage = "This patch operation contains an unexpected field and will not continue";
```

Comment:
Remove `and will not continue` I think that text is redundant as we are returning 400

Reply:
Removed